sdk/core/azure-core/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "sourceFilesToProcess": null,
+  "sourceFilesToSkip": [
+    "azure/core/internal/json/json.hpp",
+    "azure/core/internal/json/json_optional.hpp"
+  ],
+  "additionalIncludeDirectories": [
+  ],
+  "additionalCompilerSwitches": [
+    "-DBUILD_CURL_HTTP_TRANSPORT_ADAPTER",
+    "-DBUILD_TRANSPORT_WINHTTP_ADAPTER",
+    "-DCURL_STATICLIB"
+  ],
+  "includeInternal": true,
+  "includeDetail": false,
+  "includePrivate": false,
+  "filterNamespace": "Azure::",
+  "reviewName": "Azure Core Review",
+  "serviceName": null,
```

Comment:
What is `serviceName` used for in the APIView generated? Is it OK for it to be null?

Reply:
APIView doesn't seem to be upset if it's null. My guess is that it's used for a future feature in ApiView which would allow APIViews for multiple languages to be correlated. That would be keyed off of ServiceName since the ARM service name is the only constant that applies across languages.